cloudflared/umbrel-app.yml

```diff
@@ -0,0 +1,35 @@
+manifestVersion: 1
+id: cloudflared
+name: Cloudflare Tunnel client
+tagline: Access your Umbrel apps from the Internet using Cloudflare network
+icon: https://raw.githubusercontent.com/Radiokot/umbrel-cloudflared/main/static/logo.svg
+category: Networking
+version: "2023.10.0 (web 1.0.1)"
+port: 4499
+description: >-
+  Start a secure tunnel to access your Umbrel apps from the Internet using Cloudflare network. 
+  With Tunnel, you do not send traffic to an external IP — instead, 
+  this app containing a lightweight tunneluing daemon (cloudflared) creates outbound-only 
+  connections to Cloudflare's global network.
+
+
+  To use this app, you must have a set up Cloudflare account with added domains. 
+  Once you've done setting it up, check out the guide with examples to configure your own tunnel:
+  https://github.com/Radiokot/umbrel-cloudflared/wiki/How-to-set-up-Cloudflare-Tunnel-on-your-Umbrel
+
+developer: Cloudflare
+website: https://www.cloudflare.com/products/tunnel/
+submitter: Radiokot
+submission: https://github.com/Radiokot/umbrel-app-store/tree/main/radiokot-cloudflared
+repo: https://github.com/Radiokot/umbrel-cloudflared
+support: https://github.com/Radiokot/umbrel-cloudflared/issues
+gallery:
+  - https://raw.githubusercontent.com/Radiokot/umbrel-cloudflared/0f377f08ac8380ec5cd2e51702874841b465fefa/repository-assets/screen-1.png
+  - https://raw.githubusercontent.com/Radiokot/umbrel-cloudflared/0f377f08ac8380ec5cd2e51702874841b465fefa/repository-assets/screen-2.png
+  - https://raw.githubusercontent.com/Radiokot/umbrel-cloudflared/0f377f08ac8380ec5cd2e51702874841b465fefa/repository-assets/screen-3.png
```

Comment:
These can be changed to 1.jpg, 2.jpg, etc. For example:

https://github.com/getumbrel/umbrel-apps/blob/f6eae6fdc53a1bd71528fed49118e299da5d1aea/mempool/umbrel-app.yml#L25-L28

We will host the gallery images here as .jpgs:
https://github.com/getumbrel/umbrel-apps-gallery